### DISCUSSION TITLE
Dropout across multiple GPU

### DISCUSSION THREAD

**Discussion by vanshilshah97** [Lightning-AI/pytorch-lightning#19160] [DDP / multi-GPU / multi-node]
Is it necessary conceptually to set the same seed for multiple processes in a DDP paradigm to have the same dropout layer behavior across the processes for correct model training?
